.gitignore

```diff
@@ -186,3 +186,6 @@ Temporary Items
 
 # Common build tooling, C B T
 !/tools/build
+austation/code/modules/popes-xenoarchaeology/xenoarchaeology.txt
+austation/code/modules/popes-xenoarchaeology/xenoarchaeology.txt
+austation/code/modules/popes-xenoarchaeology/xenoarchaeology.txt
```

Comment:
```suggestion

# AUSTATION -- Xenoarchaeology
austation/code/modules/popes-xenoarchaeology/xenoarchaeology.txt
austation/code/modules/popes-xenoarchaeology/xenoarchaeology.txt
austation/code/modules/popes-xenoarchaeology/xenoarchaeology.txt
```

Reply:
Sorry. I think I was originally including the design doc.